src/components/Favorites/Favorites.js

```diff
@@ -1,16 +1,53 @@
 import React from 'react';
 import MovieCard from '../MovieCard/MovieCard';
+import { Link } from 'react-router-dom'
 import './Favorites.css';
+import { connect } from 'react-redux'
 
 
-const Favorites = () => {
-  const favoriteList = [<MovieCard />,<MovieCard />,<MovieCard />,<MovieCard />,<MovieCard />];
+const Favorites = ({movies, user, toggleFavorites}) => {
 
-  return (
+  if(!user.name){
+    return (
+      <div className='favorites'>
+        <h1>Please <Link to='/login'>login</Link> to save favorites.</h1>
+      </div>
```

Comment:
This is a good part of the UX. The user should log in in order to favorite a few things